lavague-server/lavague/server/channel.py

```diff
@@ -12,8 +12,15 @@
 
 
 class AgentSession(ABC):
-    uid = str(uuid.uuid4())
+    uid = ""
     agent: WebAgent
+    _stop_event: threading.Event = threading.Event()
+    _task: threading.Thread = None
```

Comment:
```suggestion
    agent: WebAgent
```
These properties are defined in constructor so we can remove them from here to avoid double creation